packages/frontend/src/services/FileUploadService.ts

```diff
@@ -0,0 +1,15 @@
+import http from 'axios';
+
+const upload = (nodeId: string, file: File): Promise<any> => {
+  let formData = new FormData();
+
+  formData.append('file', file);
+
+  return http.post(`http://localhost:3004/api/alfresco/file/hallintaraportit`, formData);
```

Comment:
Pass on the category class. Hardcoded now.

Reply:
Should we have some sort of error handling (e.g. error notification) and spinner loading also?